roles/container_runtime/defaults/main.yml

```diff
@@ -80,3 +82,5 @@ l_required_docker_version: '1.13'
 l_insecure_crio_registries: "{{ '\"{}\"'.format('\", \"'.join(l2_docker_insecure_registries)) }}"
 l_crio_registries: "{{ l2_docker_additional_registries + ['docker.io'] }}"
 l_additional_crio_registries: "{{ '\"{}\"'.format('\", \"'.join(l_crio_registries)) }}"
+
+l_crio_selinux_enabled: "{{ openshift_crio_selinux_enabled | to_json }}"
```

Comment:
I don't think an additional var is required

Reply:
What the pattern? I just copy it the from `openshift_docker_selinux_enabled`

Reply:
`l_docker_selinux_enabled` is required to be set in `daemon.json`, so we don't need it in this case. Also, docker's selinux setting is not a very good pattern code-wise